ibis/backends/pandas/execution/window.py

```diff
@@ -202,18 +202,20 @@ def trim_with_timecontext(data, timecontext: Optional[TimeContext]):
     # noop if timecontext is None
     if not timecontext:
         return data
+
     # reset multiindex and turn series into a dateframe
     df = data.reset_index()
-    # if Series dosen't contain a name, reset_index will assign
```

Comment:
Why removing?

Reply:
oh nvm. It just moved

Reply:
Not removing, just re-arranging the order.  The checks to exit this method should be put at first.